x-pack/plugins/security_solution/public/resolver/data_access_layer/factory.ts

```diff
@@ -89,7 +89,7 @@ export function dataAccessLayerFactory(
           indexPatterns,
         },
       };
-      if (category === 'alerts') {
+      if (category === 'alert') {
```

Comment:
Just wondering if the `entityType: 'alerts',` in line 97 should stay `alerts` or if it should also be `alert`